src/cfnlint/transform.py

```diff
@@ -103,7 +108,7 @@ def warning(self, message):
                     pass
 
             with WarningSuppressLogger(parser.logging):
-                sam_translator.translate(sam_template=self._template, parameter_values={})
+                self._template = sam_translator.translate(sam_template=self._template, parameter_values={})
```

Comment:
When doing a full deepcopy you need to be able to save the output of this function so you can consume it later.